bokehjs/src/coffee/models/mappers/log_mapper.coffee

```diff
@@ -20,11 +21,14 @@ export class LogMapper extends Model
     if inter_scale == 0
       value = 0
     else
-      value = (Math.log(x) - inter_offset) / inter_scale
-      if isNaN(value) or not isFinite(value)
-        value = 0
+      clamped_x = clamp(x, @source_range.start, @source_range.end)
```

Comment:
I'm not sure why, and agree it does not seem necessary and even possibly erroneous. This maps all points outside a plot region to the edge of the plot region? Seems that would produce bad results for large size markers or line points that are off screen that get moved to the edge.